test/differentiation.jl

```diff
@@ -1,7 +1,7 @@
 @testset "Differentiation" begin
     Mod.@polyvar x y
     @test differentiate(3, y) == 0
-    @test differentiate.([x, y], y) == [0, 1]
+    @test differentiate.([x, y], Ref(y)) == [0, 1]
```

Comment:
Same here, why is Ref needed ?

Reply:
Just an artefact of the time before I found the `Base.broadcastable` fix :)